setup/setup.iss

```diff
@@ -7,11 +7,14 @@
 [Setup]
 AppId={{BDD60DE7-9374-463C-8E74-8227EB03E28F}
 AppName=Dash Faction
+AppVerName=Dash Faction

```

Comment:
I don't think `AppVerName` should skip the version as this property name clearly indicates that it includes the version. `UninstallDisplayName` would resolve the problem of the uninstaller name while still keeping the version in installer